arangod/RocksDBEngine/RocksDBIndex.cpp

```diff
@@ -64,7 +64,9 @@ RocksDBIndex::RocksDBIndex(
     createCache();
   }
   RocksDBEngine* engine = static_cast<RocksDBEngine*>(EngineSelectorFeature::ENGINE);
-  engine->addIndexMapping(_objectId, this);
+  // hack to get db id and cid in RocksDBWalAccess
+  engine->addCollectionMapping(_objectId, collection->vocbase()->id(), collection->cid());
+  engine->addIndexMapping(_objectId, collection->vocbase()->id(), collection->cid(), _iid);
```

Comment:
I want a mapping from Index objectId -> Collection

Reply:
Why does the index mapping itself not suffice? It provides the collection and database IDs as well.

Reply:
You right, actually I think the main reason for changing `addIndexMapping` was to get rid of `addCollectionMapping` here, because this has the possibility of introducing collisions.
I forgot to delete the line